math.p8

```diff
@@ -1,24 +1,27 @@
-pico-8 cartridge // http://www.pico-8.com
-version 18
-__lua__
--- pico8lib math library
--- by sparr
+--- @module math
+--- Mathematical operations, mostly on numbers
+-- @author by sparr
 
 
-------------------------------------------------------------------------
--- lookup tables
-
--- a^b will overflow when a>power_overflow[b]
 local power_overflow = {32767,181,31,13,7,5,4,3,3,2,2,2,2,2} -- ,1,1,1
 
-
-------------------------------------------------------------------------
--- greatest common divisor of two numbers, recursive
-local function gcd_recursive(a, b)
- return a==0 and b or gcd_recursive(b%a, a)
-end
--- greatest common divisor of two numbers, iterative
-local function gcd(a, b)
+--- Return the greatest common divisor of two numbers
+-- This is the first of two available implementations
+-- @see gcd2
```

Comment:
I like @see here and for other multiple implementations of the same function. Not sure if I like removing the mention of recursion/iteration. Ditching it in the names might be ok, but it should still be in a comment at least.

Reply:
Agreed on all points.